### DISCUSSION TITLE
Using Bicubic instead of Lagrange?

### DISCUSSION THREAD

**Discussion by gewoonivar** [Guilouz/Creality-Helper-Script-Wiki#496] [Issues]
Is there a possibility to change to Bicubic bed mesh?

![Screenshot 2024-03-29 at 09 51 13](https://github.com/Guilouz/Creality-K1-Series/assets/134543025/ed74690d-607f-45e4-ae61-935c914b11a2)

My K1 Max (1.3.3.8) with KAMP via Helperscript is always using Lagrange, which is also under SAVE_CONFIG defaults:

![Screenshot 2024-03-29 at 09 50 12](https://github.com/Guilouz/Creality-K1-Series/assets/134543025/4c184060-3a2b-48f4-82ed-d0811f54af9d)

Where to change / modify the configuration?

**Comment by jaxxtheone** [comment]
![FireShot Capture 042 - fluidd K1 - Configuration - 192 168 10 154](https://github.com/Guilouz/Creality-K1-Series/assets/26436482/1d217365-8eb2-49a3-9ad6-082f71cee8d5)

I did add "bicubic" yesterday and it is working flawlessly. Just modify your printer.cfg for your like.

![FireShot Capture 043 - fluidd K1 - Configuration - 192 168 10 154](https://github.com/Guilouz/Creality-K1-Series/assets/26436482/44e29643-9c2b-4003-83d3-97ffafb6a794)

**Reply by gewoonivar** [reply]
Strange, adjusted my config, but doesn't seem to work for me.
Also tried probe count 7,7 but has no effect.

<img width="198" alt="Screenshot 2024-04-01 at 20 52 38" src="https://github.com/Guilouz/Creality-K1-Series/assets/134543025/d38e150d-5729-4f5c-92d0-c88808ae97f0">
<img width="440" alt="Screenshot 2024-04-01 at 20 52 27" src="https://github.com/Guilouz/Creality-K1-Series/assets/134543025/2b22ae18-e0e4-406f-bcea-d3eab7156e45">

**Comment by jaxxtheone** [comment]
All I read started on this page: 

[https://www.klipper3d.org/Bed_Mesh.html](url)

Hope it helps, maybe there's something you find that I didn't notice. I did.

**Reply by jaxxtheone** [reply]
Just redid it: Please read it from bottom to top:

16:04:24 
// Smart Park location: 10,10
16:04:24 
// Bed Mesh state has been saved to profile [kamp]
// for the current session. The SAVE_CONFIG command will
// update the printer config file and restart the printer.
16:04:24 
// Bed Mesh state has been saved to profile [default]
// for the current session. The SAVE_CONFIG command will
// update the printer config file and restart the printer.
16:04:24 
// Mesh Bed Leveling Complete
16:04:24 
// probe at 215.000,215.000 is z=-0.129520
16:04:11 
// probe at 180.000,215.000 is z=-0.152201
16:04:06 
// probe at 145.000,215.000 is z=-0.116771
16:04:00 
// probe at 110.000,215.000 is z=-0.049437
16:03:55 
// probe at 75.000,215.000 is z=0.003392
16:03:50 
// probe at 40.000,215.000 is z=0.043656
16:03:44 
// probe at 5.000,215.000 is z=0.080469
16:03:38 
// probe at 5.000,180.000 is z=0.067594
16:03:32 
// probe at 40.000,180.000 is z=0.071875
16:03:27 
// probe at 75.000,180.000 is z=-0.002687
16:03:22 
// probe at 110.000,180.000 is z=-0.056937
16:03:16 
// probe at 145.000,180.000 is z=-0.108375
16:03:11 
// probe at 180.000,180.000 is z=-0.090088
16:03:05 
// probe at 215.000,180.000 is z=-0.136312
16:02:59 
// probe at 215.000,145.000 is z=-0.118688
16:02:53 
// probe at 180.000,145.000 is z=-0.084750
16:02:48 
// probe at 145.000,145.000 is z=-0.035354
16:02:42 
// probe at 110.000,145.000 is z=-0.015688
16:02:37 
// probe at 75.000,145.000 is z=0.006740
16:02:32 
// probe at 40.000,145.000 is z=0.042906
16:02:26 
// probe at 5.000,145.000 is z=0.045125
16:02:19 
// probe at 5.000,110.000 is z=0.047188
16:02:14 
// probe at 40.000,110.000 is z=0.062460
16:02:09 
// probe at 75.000,110.000 is z=0.031156
16:02:03 
// probe at 110.000,110.000 is z=0.002125
16:01:58 
// probe at 145.000,110.000 is z=-0.022531
16:01:53 
// probe at 180.000,110.000 is z=-0.044124
16:01:47 
// probe at 215.000,110.000 is z=-0.074656
16:01:40 
// probe at 215.000,75.000 is z=-0.045312
16:01:35 
// probe at 180.000,75.000 is z=-0.001500
16:01:30 
// probe at 145.000,75.000 is z=-0.000750
16:01:24 
// probe at 110.000,75.000 is z=0.013219
16:01:19 
// probe at 75.000,75.000 is z=0.052302
16:01:13 
// probe at 40.000,75.000 is z=0.068562
16:01:08 
// probe at 5.000,75.000 is z=0.059199
16:01:01 
// probe at 5.000,40.000 is z=0.074813
16:00:56 
// probe at 40.000,40.000 is z=0.085844
16:00:50 
// probe at 75.000,40.000 is z=0.087500
16:00:45 
// probe at 110.000,40.000 is z=0.094687
16:00:40 
// probe at 145.000,40.000 is z=0.067906
16:00:34 
// probe at 180.000,40.000 is z=0.012531
16:00:29 
// probe at 215.000,40.000 is z=0.068094
16:00:22 
// probe at 215.000,5.000 is z=0.141969
16:00:17 
// probe at 180.000,5.000 is z=0.124938
16:00:11 
// probe at 145.000,5.000 is z=0.132281
16:00:06 
// probe at 110.000,5.000 is z=0.117813
16:00:01 
// probe at 75.000,5.000 is z=0.163437
15:59:55 
// probe at 40.000,5.000 is z=0.136037
15:59:50 
// probe at 5.000,5.000 is z=0.164438
